src/application/shell.ts

```diff
@@ -308,6 +308,51 @@ class ApplicationShell extends Widget {
     this._save();
   }
 
+  /*
+   * Activate the next Tab in the active TabBar.
+  */
+  activateNextTab(): void {
+    let current = this._currentTabBar();
+    if (current) {
+      let ci = current.currentIndex;
+      if (ci !== -1) {
+        if (ci < (current.titles.length-1)) {
```

Comment:
I will remove the extra parens. Can you clarify what you mean about the spaces around the binary operator?